### ISSUE TITLE
This problem will appear at night from yesterday.

### ISSUE THREAD

**Issue by 284267** [masterking32/MasterHamsterKombatBot#391]
I met this problem for the second time the next day.
![Screenshot_2024-09-09-23-02-36-510_org connectbot](https://github.com/user-attachments/assets/224b561c-4912-4283-b114-1a2ba7413012)

**Comment by copycatcodex** [comment]
Yes, me too, in the end I claimed the manual daily combo then I ran the bot again

**Comment by Fy0urM** [comment]
this has been fixed in a recent update. (I really hope so).

**Comment by 284267** [comment]
> this has been fixed in a recent update. (I really hope so).

It hasn't been fixed yet. I hope developers can fix it.

**Comment by 284267** [comment]
> Yes, me too, in the end I claimed the manual daily combo then I ran the bot again



> Yes, me too, in the end I claimed the manual daily combo then I ran the bot again

Can it run smoothly by doing this?

**Comment by Fy0urM** [comment]
> > this has been fixed in a recent update. (I really hope so).
> 
> It hasn't been fixed yet. I hope developers can fix it.

Have you checked the behavior after this PR?
https://github.com/masterking32/MasterHamsterKombatBot/pull/387

**Comment by autoattack999** [comment]
> > > this has been fixed in a recent update. (I really hope so).
> > 
> > 
> > It hasn't been fixed yet. I hope developers can fix it.
> 
> Have you checked the behavior after this PR? #387

Hallo Fy0urM ...
_My question, can the bot auto buy cards that have not been purchased to open the Daily Combo?_


![buy](https://github.com/user-attachments/assets/b519cef0-ce27-4694-a29f-102dc4a6dc10)

**Comment by Fy0urM** [comment]
> > > > this has been fixed in a recent update. (I really hope so).
> > > 
> > > 
> > > It hasn't been fixed yet. I hope developers can fix it.
> > 
> > Have you checked the behavior after this PR? #387
> 
> Hallo Fy0urM ...
> _My question, can the bot auto buy cards that have not been purchased to open the Daily Combo?_
> 
> 
> ![buy](https://github.com/user-attachments/assets/b519cef0-ce27-4694-a29f-102dc4a6dc10)
> 

The current implementation does not have this. Technically it can be done, but you have to check a lot of variables, such as whether a card is available for purchase, whether you can buy another card if the card you want is blocked by another card, whether you have to check the cooldown for each level of card upgrade, its price, and also consider that there might be cards in combos that you can't get anymore if you haven't gotten them before. So I probably won't try to implement this. I'm already ashamed for 2 days of errors and script interruptions due to my bad code.

**Comment by autoattack999** [comment]
Thank you for this very valuable information, don't be shy to give your script, you are also a human being, sometimes wrong, I am even embarrassed because I can only copy paste your script, without fully understanding. 

**_Once again, thank you for your effort and hard work!_**

**Comment by 284267** [comment]
![image](https://github.com/user-attachments/assets/e78bb00a-41cd-44ec-976a-b77a591cb52b)

"That's beautiful, I really like it. Thank you for your dedication."

**Comment by Pawansingh333** [comment]
Traceback (most recent call last):
  File "C:\Users\am\Desktop\HAMSTER\MasterHamsterKombatBot-1\main.py", line 1913, in <module>
    main()
  File "C:\Users\am\Desktop\HAMSTER\MasterHamsterKombatBot-1\main.py", line 1907, in main
    asyncio.run(RunAccounts())
  File "C:\Users\am\Desktop\HAMSTER\MasterHamsterKombatBot-1\main.py", line 1874, in RunAccounts
    account.Start()
  File "C:\Users\am\Desktop\HAMSTER\MasterHamsterKombatBot-1\main.py", line 1734, in Start
    self.ClaimDailyCombo()
  File "C:\Users\am\Desktop\HAMSTER\MasterHamsterKombatBot-1\main.py", line 581, in ClaimDailyCombo
    isClaimed = buyResult.get("dailyCombo", {}).get("isClaimed", False)
AttributeError: 'NoneType' object has no attribute 'get'

**Comment by Fy0urM** [comment]
> Traceback (most recent call last): File "C:\Users\am\Desktop\HAMSTER\MasterHamsterKombatBot-1\main.py", line 1913, in main() File "C:\Users\am\Desktop\HAMSTER\MasterHamsterKombatBot-1\main.py", line 1907, in main asyncio.run(RunAccounts()) File "C:\Users\am\Desktop\HAMSTER\MasterHamsterKombatBot-1\main.py", line 1874, in RunAccounts account.Start() File "C:\Users\am\Desktop\HAMSTER\MasterHamsterKombatBot-1\main.py", line 1734, in Start self.ClaimDailyCombo() File "C:\Users\am\Desktop\HAMSTER\MasterHamsterKombatBot-1\main.py", line 581, in ClaimDailyCombo isClaimed = buyResult.get("dailyCombo", {}).get("isClaimed", False) AttributeError: 'NoneType' object has no attribute 'get'

update to the last version
